### PR TITLE
Update category-porn

### PR DESCRIPTION
**Pull request by bijoni**

### CODE REVIEW

File: data/category-porn
Comment (rootmelo92118) [inline data/category-porn:3385]: 如果這個域名跟上面幾個域名是相似的可以幫忙歸類一下。

According to your submission, I think you can speak Chinese. Please tell me if I have misunderstanding for it.

**Comment by bijoni** [comment]
How about this?

**Comment by rootmelo92118** [comment]
> How about this?

Great, but another maintainer requested to review this pull request later. We should wait for him until the review is finished.

**Review by IceCodeNew — APPROVED** [review]

**Comment by IceCodeNew** [comment]
Sorry, I was reviewing on a separate commit, and the PR itself turns out does not meet the requirement for merging.
I will fix it in #1603

**Pull request merged**
